src/renderer/containers/ReplayBrowser/FileSelectionToolbar.tsx

```diff
@@ -53,13 +57,12 @@ export const FileSelectionToolbar: React.FC<FileSelectionToolbarProps> = ({
           {totalSelected} files selected
         </div>
         <div>
-          <Button
-            color="secondary"
-            variant="contained"
-            size="small"
-            onClick={() => setShowDeletePrompt(true)}
-            startIcon={<DeleteIcon />}
-          >
+          <DraggableFiles fullPaths={filePaths}>
+            <Button color="secondary" variant="contained" size="small" startIcon={<DragIndicatorIcon />}>
+              Drag & Drop Replays
```

Comment:
```suggestion
              Share
```

This is a bit long for the button and doesn't really tell the user what the usage it has. Maybe `Share` would work better.